Terminal.Gui/Core/View.cs

```diff
@@ -3155,5 +3158,45 @@ public View GetTopSuperView (View view = null, View superview = null)
 
 			return top;
 		}
+
+		/// <summary>
+		/// Finds which view that belong to the <paramref name="start"/> at the provided location.
```

Comment:
I believe all coordiantes are  elative to `start`'s superview, right? If that's correct please add this detail to the API docs. If not please add the right verbiage.